README.md

```diff
@@ -47,6 +47,17 @@ make collect-all # default collect folder is cpu2006_build_$(TIMESTAMP)
 make collect-all ELF_PATH=/path/to/elf # ELF_PATH is optional
 ```
 
+# With Vector extension
+
+- compile binarys
+```
+make ARCH=riscv64 \
+     CROSS_COMPILE=riscv64-unknown-linux-gnu- \
+     OPTIMIZE="-O3 -flto -march=rv64gcv_zvl128b_zba_zbb_zbc_zbs -ftree-vectorize  -mabi=lp64d -mrvv-max-lmul=m4 -mrvv-vector-bits=zvl" \
+     SUBPROCESS_NUM=5 \
+     build-all -j 29
```

Comment:
why j 29 here? Maybe we don't need to specify `-j` ? or specify something like `nproc`?

Reply:
-  '-j 29' means that 29 subitems are constructed at the same time
-  'SUBPROCESS_NUM' means how many threads are used in the construction of each individual subitem